src/trackers/LT.py

```
@@ -63,17 +63,51 @@ async def get_name(self, meta):
+                lang = audio.get("Language", "").lower()
+                title = str(audio.get("Title", "")).lower()
+
+                if "commentary" in title:
+                    continue
+
+                # Check if title contains keywords
+                is_latino_title = any(kw in title for kw in latino_keywords)
+                is_castilian_title = any(kw in title for kw in castilian_keywords)
+
+                # 1. Check strict Latino language codes or Edge Case: Language is 'es' but Title contains Latino keywords
+                if lang in audio_latino_check or lang == 'es' and is_latino_title:
```

Ruff (0.14.8)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)